jekyll/_cci2/test.md

```diff
@@ -1,51 +1,88 @@
+To automatically run your test suites in a project pipeline, you will add configuration keys in your `.circleci/config.yml` file. These would typically be defined as a **step** or collection of steps to be executed in a **job**.
 
-## Deploy
-{: #deploy }
+A pipeline might consist of a single workflow, with a single job defined that includes a step to execute a suite of tests within an execution environment.
 
-Refer to the following document for information and examples of deployment targets and tools.
+```yaml
+jobs:
+  build-and-test:
+    docker:
+      - image: cimg/node:16.10
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          name: Run tests
+          command: npm test
+```
 
-Document | Description
-----|----------
-<a href="{{ site.baseurl }}/2.0/deployment-integrations/">Deployment</a> | Configure automated deployment to AWS, Azure, Firebase, Google Cloud, Heroku, NPM, or virtually any other service.
-<a href="{{ site.baseurl }}/2.0/artifactory/">Artifactory</a> | Configure automated uploads to Artifactory with the Jfrog CLI.
-<a href="{{ site.baseurl }}/2.0/packagecloud/">packagecloud</a> | Publish packages to packagecloud.
-{: class="table table-striped"}
+`run` is a built-in step that runs commands in a shell. To read more about the `run` step for executing tests and other commands, go to the [Configuring CircleCI]({{ site.baseurl }}/2.0/configuration-reference) reference.
 
-We’re thrilled to have you here. Happy building!
+Depending on your requirements, you might have more complex workflows that orchestrate multiple jobs. For example, you might have several concurrent jobs for building and testing your project in separate Linux, macOS, and Windows execution environments. You might also want to require that a test job is run only if a preceding build job is successful.
+
+For a more in-depth look at workflows functionality, read the [Workflows]({{ site.baseurl }}/2.0/workflows) document. You can also refer to the [Sample Configuration]({{ site.baseurl }}/2.0/sample-config) page for more workflow examples.
+
+## Use orbs to simplify testing
+{: #orbs }
+
+Orbs provide an easy way to integrate popular testing tools into your configuration. You can invoke CircleCI partner orbs such as Cypress, LambdaTest, and Sauce Labs in your `.circleci/config.yml` file. That will then allow you to include common testing tasks in your pipeline by running concise usage commands in your jobs.
+
+Orbs are available for mobile testing, browser testing, load testing, and code coverage. To get started with orbs, refer to the [Orbs Introduction]({{ site.baseurl }}/2.0/orb-intro) document. To view the orb registry, visit the [CircleCI Developer Hub](https://circleci.com/developer/orbs?query=&category=Testing).
+
+## Store test data
+{: #store-test-data }
+
+Results from testing can be saved in CircleCI by adding the following steps within a job:
+
+### `store_test_results`
```

Comment:
Could you consider adding  a little example of using this step rather than using the step syntax itself as the heading? I feel like it looks odd in the rendered output.

Also I think there's quite a but of info on this step in Collecting Test Data, which you reference in the following section but not this one.

Reply:
I've changed the layout a bit so that these two steps aren't their own headings anymore, but under **Store test data**, and the bit about Collecting Test Data is also now just part of that same section.